src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java

```diff
@@ -87,6 +87,7 @@ protected Employee toEntity(EmployeeDto dto) {
         employee.setTechnologies(technologies);
         employee.setPaymentsInformation(dto.paymentInformation() == null ? Collections.emptySet()
                 : dto.paymentInformation().stream().map(PaymentInformation::new).collect(Collectors.toSet()));
+        employee.setActive(true);
```

Comment:
all the employees that are created will be active? @agustinb4 . Will need a criteria to set true in active field and query to verify the rest of employees active value

Reply:
Added active switch to address this
![image](https://github.com/entropy-code/entropay-employees/assets/46864825/3ae05b0a-83e9-47f4-a491-e6bf70639801)